content/kit/_index.md

```diff
@@ -28,19 +28,24 @@ In addition to the boards, your kit will also contain:
 
 | Part                         | Quantity | Specification               | Notes                                  |
 |------------------------------|----------|-----------------------------|----------------------------------------|
+| Battery Bag                  | 1        |                             |                                        |
 | Battery Charger              | 1        | iMAX B6                     |                                        |
 | Battery Charger Power Supply | 1        | 12V 5A                      |                                        |
 | Battery                      | 1        | 11.1V 2.2Ah Lithium Polymer |                                        |
```

Comment:
I thought we were giving them 2 batteries? :)

Reply:
"Battery Bag"

Reply:
But next to battery there is quantity one?